examples/kubernetes/block-volume/README.md

```diff
@@ -0,0 +1,39 @@
+## Raw Block Volume
+This example shows how to consume a dynamically provisioned EBS volume as a raw block device.
+
+### Edit [Persistence Volume Claim Spec](./specs/raw-claim.yaml)
+Make sure the `volumeMode` is `Block`.
+
+### Edit [Application Pod](./specs/pod.yaml)
+Make sure the pod is consuming the PVC with the defined name and `volumeDevices` is used instead of `volumeMounts`.
+
+### Deploy the Application
+```sh
+>> kubectl apply -f examples/kubernetes/block-volume/specs/storageclass.yaml
```

Comment:
For this bash example, since it doesn't have an output can you remove `>>` to make it easier to copy and paste.